lib/Dialect/Torch/Transforms/ReifyAbstractInterpCalculationsUtils.cpp

```diff
@@ -76,9 +87,17 @@ LogicalResult Torch::wrapWithCalculateOpIfLibraryFunctionAvailable(
   std::string libFuncName =
       (getLibraryFunctionPrefix(libFuncKind) + Twine(name)).str();
   auto libFunc = library.lookupSymbol<func::FuncOp>(libFuncName);
-  if (!libFunc)
-    return success();
-  libFuncNamesUsed.push_back(libFuncName);
+  if (!libFunc) {
+    auto parentModule = op->getParentOfType<ModuleOp>();
+    if (parentModule)
+      libFunc =
+          op->getParentOfType<ModuleOp>().lookupSymbol<func::FuncOp>(libFuncName);
+    if (!libFunc)
+      return success();
+  } else {
+    libFuncNamesUsed.push_back(libFuncName);
```

Comment:
`libFuncNamesUsed` is ultimately used to "import" shape functions - since the user-provided shape functions are already in the user module, this isn't necessary (and causes a segfault somewhere around [ReifyAbstractInterpCalculationsUtils.cpp#L159](https://github.com/llvm/torch-mlir/blob/cc5e2c230c576eca7210a4e7f475e1e9bcd965b3/lib/Dialect/Torch/Transforms/ReifyAbstractInterpCalculationsUtils.cpp#L159).